docs/user-guide/annotations.md

```diff
@@ -348,7 +348,7 @@ The annotation `nginx.ingress.kubernetes.io/ssl-passthrough` allows to configure
 **Important:**
 
 - Using the annotation `nginx.ingress.kubernetes.io/ssl-passthrough` invalidates all the other available annotations. This is because SSL Passthrough works in L4 (TCP).
-- The use of this annotation requires the flag `--enable-ssl-passthrough` (By default it is disabled)
+- The use of this annotation requires proxy protocol to be enabled in load balancer. For example enabling proxy protocol for AWS ELB described [here](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-proxy-protocol.html). If you're using ingress-controller without load balancer then the flag `--enable-ssl-passthrough` required (By default it is disabled).
```

Comment:
"in _the_ load-balancer"
"_P_ roxy _P_ rotocol"
"for AWS ELB _is_ described"
"flag `--enable-ssl-passthrough`_is_ required"
"_b_ default"

Reply:
done.